cmd/dm-master/main.go

```diff
@@ -71,3 +77,8 @@ func main() {
 
 	log.Info("dm-master exit")
 }
+
+func main() {
```

Comment:
not used anymore?

Reply:
main function is still the execution entrance of `dm-master`, `dm-worker`